Packs/MajorBreachesInvestigationandResponse/pack_metadata.json

```diff
@@ -117,6 +117,10 @@
         "CVE_2024_47575": {
             "mandatory": false,
             "display_name": "CVE-2024-47575 - FortiManager Authentication Bypass"
+        },
+        "CVE_2025_31324": {
+            "mandatory": false,
+            "display_name": "CVE-2025-31324 - SAP NetWeaver Visual Composer"
         }
     },
     "marketplaces": [
```

Comment:
Is rapid should be on all platforms as well?

Reply:
yes